### ISSUE TITLE
detection of arm64 CPU features on FreeBSD

### ISSUE THREAD

**Issue by cperciva** [Tarsnap/libcperciva#410]
We're using the linux getauxval API but not the FreeBSD elf_aux_info equivalent.  We should do that...

**Comment by gperciva** [comment]
I've had that branch for months, but you told me not to bother until EC2 on t4g was more stable!

**Comment by cperciva** [comment]
Right, I think that was before we shipped 13.0.  Let's merge those bits now.

**Comment by cperciva** [comment]
This is done now, right?
